button/popup.css

```diff
@@ -292,11 +296,11 @@ http://matthewlein.com/ceaser/
 }
 
 .sitetitle {
-    margin-top: 3px;
     display: block;
-    left: -10px;
+    height: 35px;
     position: inherit;
-    padding: 10px 0 0 15px;
+    padding: 15px 10px 0 5px;
```

Comment:
![Codacy](https://www.codacy.com/versioned/images/favicon.png) Issue found: [Using height with padding can sometimes make elements larger than you expect.](https://www.codacy.com/app/tomastaro/notemark/file/1772214731/issues/source?bid=2884821&fileBranchId=3065795#l302)